plugin/code_actions.py

```diff
@@ -82,15 +82,16 @@ def request_for_region_async(
         self,
         view: sublime.View,
         region: sublime.Region,
-        session_buffer_diagnostics: Sequence[Tuple[SessionBufferProtocol, Sequence[Diagnostic]]],
+        session_buffer_diagnostics: List[Tuple[SessionBufferProtocol, List[Diagnostic]]],
         actions_handler: Callable[[CodeActionsByConfigName], None],
-        only_kinds: Optional[Dict[str, bool]] = None
+        only_kinds: Optional[Dict[str, bool]] = None,
+        manual: Optional[bool] = False,
```

Comment:
Since you provide a boolean default value already, and there isn't any special handling for `None`, I guess this argument type doesn't need to be Optional.

Reply:
yes, true